onionshare_gui/tab/server_status.py

```diff
@@ -279,15 +279,6 @@ def update(self):
                     )
                 else:
                     self.server_button.setText(strings._("gui_please_wait"))
-
-                if self.settings.get("general", "autostart_timer"):
```

Comment:
This might look weird in a diff but in fact it was a duplicate chunk of code here that isn't needed anymore due to the change in the conditional above.